docs/swupd.1.rst

```diff
@@ -228,6 +228,11 @@ SUBCOMMANDS
         Do not perform an update, instead download all resources needed
         to perform the update, and exit.
 
+    - `-m, --migrate`
```

Comment:
We should actually reserve `-m` for specifying the version to update to (#257)

Reply:
Are we expecting people to do swupd update -m 18850?

Reply:
Yes, according to discussion in #257, there is demand for that functionality.

Reply:
No update on #257 yet, but I think the team decided to move forward with implementing a `-m` option for swupd update. In this PR, @tmarcu added a `-T` option instead.

Reply:
Looks like the documentation still needs to be updated though.

Reply:
/me looks at the source line we are commenting on... Yes, this .rst file should be updated.